backend/examples/verify_inclusion_on_local.rs

```diff
@@ -0,0 +1,61 @@
+#![feature(generic_const_exprs)]
```

Comment:
I suggest to remove `verify_inclusion_on_local` from the examples and keep `verify_inclusion_on_contract` as the verification method by default

Reply:
It make sense, removed at [c9643c9](https://github.com/summa-dev/summa-solvency/pull/159/commits/c9643c948ebcdae61ba46f72fde841acb6710233)